sap/note/ini.go

```diff
@@ -304,7 +304,7 @@ func (vend INISettings) Apply() error {
 			continue
 		}
 
-		if revertValues && vend.SysctlParams[param.Key] != "" {
+		if revertValues && vend.SysctlParams[param.Key] != "PNA" {
```

Comment:
Why PNA?

Reply:
PNA == Parameter not available. 
This if-statement was the last left over from a change made for saptune 3.1 years ago. I simply overlooked it and just found it during my hunt of the missing bytes :smile: